test/test_artifacts/v2/scripts/run_pysdk_tests.sh

```diff
@@ -8,13 +8,44 @@ git checkout tags/v$pysdk_version
 
 # Install test dependencies of sagemaker-python-sdk
 # Using pip as some of the packages are not available on conda-forge
-pip install -r requirements/extras/test_requirements.txt
+pip install --use-deprecated=legacy-resolver -r requirements/extras/test_requirements.txt
```

Comment:
What does `--use-deprecated=legacy-resolver` do?

Reply:
yeah so without using it, getting the following error :
```
The conflict is caused by:
onnx 1.15.0 depends on protobuf>=3.20.2
tensorflow 2.12.0 depends on protobuf!=4.21.0, 1=4.21.1, 1=4.21.2, 1=4.21.3, 1=4.21.4, 1=4.21.5, <5.0.0dev and >=3.20.3
mlflow 2.12.2 depends on protobuf<5 and >=3.12.0
tensorboard 2.12.0 depends on protobuf>=3.19.6
opentelemetry-proto 1.28.0 depends on protobuf<6.0 and >=5.0
```

so for opentelemetry-proto 1.28.0, protobuf version should be >=5.0 and the current protobuf Version: 4.25.3

To resolve this, flag ```--use-deprecated=legacy-resolver``` is being added.